HW0/P6/P6B.py

```diff
@@ -16,17 +16,27 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [10**-6, 10**-5,10**-4,10**-3,10**-2,10**-1,1]
 
     for t in wait_time:
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
+	# serially
+	ts_start = time.time()
+	for i in range(16):
+	    burnTime(t)
+	ts_end = time.time()
 
-        # Compute the ratio of these times
-        # ratio.append(serialTime/parallelTime)
+        serialTime = ts_end - ts_start
+
+	tp_start = time.time()
+	result = pool.map(burnTime, [t]*16)	
+	tp_end = time.time()
```

Comment:
I'm not sure if this is relevant, but the times of initialization of the range(16) list vs. [t]*16 may be different and could potentially impact the results. I would look into this to check if these are different, or this could be avoided better by initializing this before running time.time().